### PR TITLE
Add RIP: Multi-dimensional 256-bit nonce for RIP-7560 Account Abstraction transactions

### PR DESCRIPTION
**Pull request by forshtat**

### CODE REVIEW

File: RIPS/rip-9996.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+---
+rip: 0
```
Comment (abcoathup) [inline RIPS/rip-9996.md:2]:
```suggestion
rip: 7712
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

File: RIPS/rip-9996.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+title: Enable RIP-7560 transactions using a two-dimensional nonce
+description: An RIP-7560 transaction modification that allows Smart Contract Accounts to define their own transaction sequencing
+author: Vitalik Buterin (@vbuterin), Yoav Weiss (@yoavw), Alex Forshtat (@forshtat), Dror Tirosh (@drortirosh), Shahaf Nacson (@shahafn)
+discussions-to:
```
Comment (abcoathup) [inline RIPS/rip-9996.md:6]: Please add a topic in Ethereum Magicians forum with a link to this PR for discussion.

**Review by yoavw — APPROVED** [review]

**Review by gatleas17 — COMMENTED** [review]
Thanks

**Review by josephjamesmorris700-hub — COMMENTED** [review]
Working excellently

**Review by MichealSD13 — APPROVED** [review]

**Pull request merged**